src/webapi/error.rs

```diff
@@ -36,6 +36,26 @@ pub trait IError: ReferenceType {
 #[reference(instance_of = "Error")]
 pub struct Error( Reference );
 
+impl Error {
+    /// Creates a new `Error` with the specified `description`.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)
+    #[inline]
+    pub fn new( description: &str ) -> Self {
+        js!( return new Error( @{description} ); ).try_into().unwrap()
+    }
+
+    /// Prints the `Error` to the console (this prints the error's description and also its stack trace).
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Console/error)
+    #[inline]
+    pub fn print( &self ) {
```

Comment:
That's fair, though I had kind of figured that `console.error(e)` was a placeholder, and we might implement better error handling in the future, in which case having `e.print()` would be nice. But I don't have a particularly strong opinion about it, so I don't mind removing the `print` method.

Reply:
Since we don't currently have a `console::error` function, I'd rather leave this in for now. Afterwards I'll make another pull request which adds in `console::error` and removes `print`